autoprofile/auto_profiler.go

```diff
@@ -34,7 +36,14 @@ type AutoProfiler struct {
 	GetExternalPID func() string
 }
 
-func NewAutoProfiler() *AutoProfiler {
+func GetProfiler() *AutoProfiler {
```

Comment:
Since the use of `Get` is generally discouraged in Go, can we name it just `Profiler()` or maybe even `DefaultProfiler`?